app/controllers/kudos_controller.rb

```diff
@@ -0,0 +1,66 @@
+class KudosController < ApplicationController
```

Comment:
polecalbym Ci nie uzywac na poczatku scaffoldow - ucinasz sobie praktyke pisania tego tekstu / kodu, a to istotne - szczegolnie na poczatku :)

Reply:
tutaj był niezły temat wiesz, tylko się nie złość
bo na początku naprawdę napisałem model i sobie zacząłem na nim działa
potem mi się przypomniało, że jest coś takiego jak scaffoldow i pomyślałem, jest funkcjonalność która postawi mi sporo za jednym zamachem to czemu miałbym z niej nie skorzystać...
tym bardziej jak potem wrzuciłeś na nerds.family kolejne zadanie i zobaczyłem, że będzie trzeba dopisać edit i destroy, a przecież w scaffoldow masz cały CRUD, nawet w kursach które mi podsyłałeś tak się to robiło
przepraszam, prześledzę zatem te struktury po prostu teraz z uwagą

Reply:
Nic sie nie stalo - caly drugi sprint bedziesz utrwalac scaffoldy robiac admin panel ✌️

Reply:
uff, dzięki - już się nie mogę doczekać :P